prac_07/myguitars.py

```diff
@@ -0,0 +1,66 @@
+import csv
+from guitar import Guitar
+
+
+def main():
+    guitars = load_guitars("guitars.csv")
+    display_guitars(guitars)
+
+    print("\n--- Sorted Guitars by Year ---")
+    guitars.sort()
+    display_guitars(guitars)
+
+    add_new_guitars(guitars)
+    save_guitars("guitars.csv", guitars)
+
+
+def load_guitars(filename):
+    """Read guitar data from a CSV file and return a list of Guitar objects"""
+    guitars = []
+    try:
+        with open(filename, "r") as file:
+            reader = csv.reader(file)
+            for row in reader:
+                name, year, cost = row[0], int(row[1]), float(row[2])
+                guitars.append(Guitar(name, year, cost))
+    except FileNotFoundError:
+        print(f"File '{filename}' not found. Starting with an empty list.")
+    return guitars
+
+
+def display_guitars(guitars):
+    """Display the list of guitars."""
+    for guitar in guitars:
+        print(guitar)
+
+
+def add_new_guitars(guitars):
+    """Prompt the user to enter new guitars and add them to the list."""
+    print("\nEnter new guitars: ")
+    name = input("Guitar Name: ")
+
+    while name:
```

Comment:
I think it's better to use this here： while name !='':
This is how we use it on Programming Patterns